common/lib/xmodule/xmodule/js/spec/video/video_control_spec.js

```diff
@@ -278,7 +278,8 @@
 
         describe('constructor with end-time', function () {
             it(
-                'saved position is 0, timer slider and VCR set to 0:00',
+                'saved position is 0, timer slider and VCR set to 0:00 ' + 
```

Comment:
The version tests something a bit different, and I wanted to reflect this. The original code tested the behavior that the end-time parameter didn't influence the video's end time. However, with the new functionality, the video's end time is the end-time parameter.